tests/hotkey.spec.ts

```diff
@@ -37,13 +37,12 @@ test('rich-text hotkey scope on single press', async ({ page }) => {
   await assertRichTexts(page, ['\n']);
 });
 
-//TODO FIXME: select all feature is modified
-test.skip('single line rich-text inline code hotkey', async ({ page }) => {
+test('single line rich-text inline code hotkey', async ({ page }) => {
   await enterPlaygroundRoom(page);
   await initEmptyParagraphState(page);
   await focusRichText(page);
   await page.keyboard.type('hello');
-  await selectAllByKeyboard(page);
+  await dragBetweenIndices(page, [0, 0], [0, 5]);
```

Comment:
@JimmFly we don't have to skip all these test cases 😄

Reply:
> @JimmFly we don't have to skip all these test cases 😄

Sorry, I didn't read carefully the content of this test.